Directory.Packages.props

```diff
@@ -92,7 +92,7 @@
     <PackageVersion Include="Roslyn.Diagnostics.Analyzers"                                           Version="3.11.0-beta1.23472.1" />
 
     <!-- NuGet -->
-    <PackageVersion Include="NuGet.VisualStudio"                                                     Version="6.0.0-preview.3.158" />
+    <PackageVersion Include="NuGet.VisualStudio"                                                     Version="17.11.0-zlocal.32767" />
```

Comment:
I don't have write access to the feed in your nuget.config file, so CI is failing because NuGet.VisualStudio version 17.11.0-preview.2.61 can't be restored.

Can someone either click the button in Azure Artifact's UI, or restore my branch locally, so this package version gets added to your feed.